src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
```

Comment:
why do we need a lot of manipulations? Why just not get all `tr` and clone what we need with the loop